website/aibot/gemini_api.py

```
@@ -0,0 +1,82 @@
+
+    Returns:
+        Optional[List[float]]: The embedding vector, or None if an error occurs.
+    """
+    try:
+        response = g_client.models.embed_content(
+            model=embedding_model,
+            contents=text,
+            config=types.EmbedContentConfig(output_dimensionality=settings.QDRANT_VECTOR_SIZE, task_type=task_type),
+        )
+        [embeddings] = response.embeddings
+        return embeddings.values
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)